rdr_service/tools/tool_libs/consents.py

```diff
@@ -44,6 +61,31 @@ def run(self):
             self.upload_records()
         elif self.args.command == 'check-retro-sync':
             self.check_retro_sync()
+        elif self.args.command == 'retro-validation':
+            self.retro_validate()
+
+    def _call_server_for_retro_validation(self, participant_ids):
+        if len(participant_ids) > 0:
+            print(f'Processing batch that starts with {participant_ids[0]}')
+
+        response = requests.post(
+            'https://offline-dot-all-of-us-rdr-prod.appspot.com/offline/ManuallyValidateFiles',
```

Comment:
What determines if we define/read endpoints from the system config vs. hardcoding?